UnitsNet/GeneratedCode/Quantities/VolumeFlow.g.cs

```diff
@@ -1153,6 +1153,25 @@ public override bool Equals(object obj)
             return AsBaseUnitCubicMetersPerSecond().Equals(((VolumeFlow) obj).AsBaseUnitCubicMetersPerSecond());
         }
 
+        /// <summary>
+        ///     Compare equality to another VolumeFlow within the given absolute or relative tolerance.
+        ///     Relative tolerance is when the difference between the two quantities is not greater than the scale of the values compared to the tolerance.
+        ///     Absolute tolerance is when the difference between the two quantities is not greater than exactly the tolerance.
+        ///     Note that it is advised against specifying zero difference, due to the nature
+        ///     of floating point operations and using System.Double internally.
+        /// </summary>
+        /// <param name="other">The other quantity to compare to.</param>
+        /// <param name="tolerance">The comparison tolerance.</param>
+        /// <param name="comparisonType">The comparison type: either relative or absolute.</param>
+        /// <returns>True if the difference between the two values is not greater than the specified relative or absolute tolerance.</returns>
+        public bool Equals(VolumeFlow other, double tolerance = 0.00001, ComparisonType comparisonType = ComparisonType.Relative)
```

Comment:
Tolerance probably needs to be of type `VolumeFlow`, since the consumer may be working with units of entirely different orders of magnitude than the base unit - so the default of `0.00001` may be way too small or way too large. 

A second option is to rely on the fact that we keep track of the `Unit` the quantity was constructed with, so we could pick one of the quantities' `Unit` and convert both quantities to a numeric value of that unit, then compare that with a default numeric tolerance. So when comparing a 1 nanometer with 2 nanometers, then the tolerance would 0.000001 nanometers by default rather than 0.000001 meters. The pitfall here is that it matters WHICH value you choose the unit for, as you get very different behavior when comparing 1 kilometer to 1e12 nanometers vs comparing 1e12 nanometers to 1 kilometer. In this particular example, I would expect `this` to be the reference - which I guess is intuitive.

Reply:
I think I like option #2 better. Absolute/relative can be a bit confusing and keeping it a double makes it easier to think of as a "percentage". Being typed would make sense for absolute differences, but since this is a generic method I think it's better as a double.

I made this in the same thinking of String.Equals where you can pass in the comparison type (ignore case, etc.).

I agree the reference being [this] is intuitive.